formats/fatality_notice/frontend/examples/fatality_notice.json

```diff
@@ -0,0 +1,67 @@
+  "locale": "en",
+  "updated_at": "1881-03-23T14:00:00.023Z",
+  "public_updated_at": "1881-05-17T14:00:00.023Z",
+  "details": {
+    "body": "<div class=\"govspeak\">\n<h2 id=\"sir-george-pomeroy-colley\">Sir George Pomeroy Colley</h2>\n<figure class=\"image embedded\">\n<div class=\"img\"><img alt=\"Photograph of Sir George Pomeroy Colley posed standing\" src=\"https://assets.digital.cabinet-office.gov.uk/media/573f2c49ed915d05fd000000/colley.jpg\"></div>\n<figcaption>Sir George Pomeroy Colley (All rights reserved.)</figcaption>\n</figure>\n<p>Colley served nearly all of his military and administrative career in British South Africa, but he played a significant part in the Second Anglo-Afghan War as military secretary and then private secretary to the governor-general of India, Lord Lytton. The war began in November 1878 and ended in May 1879 with the Treaty of Gandamak.</p>\n<p>After the war Colley returned to South Africa, became high commissioner for South Eastern Africa in 1880... and died a year later at the Battle of Majuba Hill during the First Boer War.</p>\n<p>A british officer had the following to say</p>\n<blockquote>\n  <p>Major General Colley was an exceptional talent, and it is with great sadness that we have learned about this loss. His contribution to Britain through his efforts in the Boer War will not be forgotten.</p>\n</blockquote>\n</div>",
+    "first_public_at": "1881-03-23T14:00:00.023Z",
+    "government": {
+      "title": "1880 to 1885 Liberal government",
+      "slug": "1880-to-1885-liberal-government",
+      "current": false
+    },
+    "political": false,
+    "emphasised_organisations": ["d994e55c-48c9-4795-b872-58d8ec98af12"],
+    "change_history": [
+      {
+        "public_timestamp": "1881-05-17T14:00:00.023Z",
+        "note": "Add comment from British officer"
+      },
+      {
+        "public_timestamp": "1881-03-23T14:00:00.023Z",
+        "note": "First published."
+      }
+    ]
+  },
+  "links": {
+    "organisations": [
+      {
+        "content_id": "d994e55c-48c9-4795-b872-58d8ec98af12",
+        "title": "Ministry of Defence",
+        "base_path": "/government/organisations/ministry-of-defence",
+        "api_url": "https://www.gov.uk/api/content/government/organisations/ministry-of-defence",
+        "web_url": "https://www.gov.uk/government/organisations/ministry-of-defence",
+        "locale": "en",
+        "analytics_identifier": "D17"
+      }
+    ],
+    "ministers": [
+      {
+        "content_id": "8537348f-c0f1-11e4-8223-005056011aef",
+        "title": "William Ewart Gladstone",
+        "base_path": "/government/people/william-ewart-gladstone",
+        "api_url": "https://www.gov.uk/api/content/government/people/william-ewart-gladstone",
+        "web_url": "https://www.gov.uk/government/people/william-ewart-gladstone",
+        "locale": "en"
+      }
+    ],
+    "field_of_operation": [
+      {
+        "content_id": "8537348f-c0f1-11e4-9312-005056011aef",
```

Comment:
This should probably be a different content id to the minister to avoid confusion